README.md

```diff
@@ -1,4 +1,32 @@
-# user-feedback
+# User Feedback
```

Comment:
Can we include the word DAFNI in here somewhere prominently? Make it look less template-y

Reply:
Done!